demosplan/DemosPlanCoreBundle/Logic/FileResponseGenerator/PdfResponseGenerator.php

```diff
@@ -34,10 +36,12 @@ public function __invoke(array $file): Response
         $response = new Response($file['content'], 200);
         $response->headers->set('Pragma', 'public');
         $response->headers->set('Content-Type', 'application/pdf; charset=utf-8');
-        $response->headers->set(
-            'Content-Disposition',
-            $this->nameGenerator->generateDownloadFilename($file['filename'])
-        );
+        if ($this->nameGenerator) {
```

Comment:
you can skip the if test when you inject the service

Reply:
Sorry, I forgot to remove that